app/models/mixins/cockpit_support_mixin.rb

```diff
@@ -0,0 +1,33 @@
+module CockpitSupportMixin
+  extend ActiveSupport::Concern
+
+  included do
+    supports :cockpit_console do
+      message = 'The web-based console is not available because the'
+      if respond_to?(:platform) && platform == 'windows'
+        unsupported_reason_add(:cockpit_console, _("#{message} Windows platform is not supported"))
+      end
+      unsupported_reason_add(:cockpit_console, _("#{message} 'Cockpit' role is not enabled")) unless MiqRegion.my_region.role_active?('cockpit_ws')
```

Comment:
The Cockpit Role does not exist on Euwe to be enabled.  I believe that role was created in #12506.